src/pages/index.tsx

```diff
@@ -70,20 +65,17 @@ const Home: NextPage<DefaultSeoInfo> = (props) => {
 	 */
 
 	const renderPageContent = () => {
-		if (!hasCheckedLogin && isBrowser()) {
-			return <Loading fullscreen owner="root index page" />;
+		if (isContentPageLoading) {
+			return <Loading fullscreen owner="slug page: render page content" />;
 		}
-		if (isLoggedIn && !!user) {
-			if (showLinkedSpaceAsHomepage && linkedSpaceSlug) {
-				return <Loading fullscreen owner="root page logged" />;
-			}
-			return <LoggedInHome {...props} />;
+		if (contentPageInfo) {
+			return <ContentPageRenderer path={'/' as string} userGroupId={user?.groupId} />;
```

Comment:
In the latest version of the admin-core-ui, loading a content page using just the path is no longer supported:
https://github.com/viaacode/react-admin-core-module/blob/release/archief2fase2/ui/src/react-admin/modules/content-page/components/ContentPageRenderer/ContentPageRenderer.tsx#L24-L29

You can replace this by:
```suggestion
			return <ContentPageRenderer contentPageInfo={contentPageInfo} />;
```